tests/kfs_open_test.c

```diff
@@ -13,48 +14,63 @@ void kfs_open_teardown(void) {
 }
 
 START_TEST(kfs_open_exist) {
-    int res = creat("exist.txt", 0777);
-    ck_assert_int_ne(res, 0);
-    char *path = str_concat(LOCAL_DISC_CACHE_PATH, "exist.txt");
-    FILE *file = kfs_open(path, O_RDWR);
-    fail_if(file == NULL);
+    char * path = strcat(LOCAL_DISC_CACHE_PATH, "exist.txt");
+    struct fuse_file_info create = {O_CREAT};
+    int res = kfs_create(path, 0777, &create);
+    ck_assert_int_eq(res, 0);
+    struct fuse_file_info open = {O_RDONLY|O_WRONLY};
+    res = kfs_open(path, &open);
```

Comment:
check create.fh is not null

Reply:
added line fail_if(&create.fh == NULL) to check create filehandle